docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.horizontal-card.scss

```diff
@@ -0,0 +1,144 @@
+.horizontal-card {
+  display: flex;
+  flex-direction: column;
+
+  @include grid-media-min("md") {
+    flex-direction: row;
+  }
+
+  @include grid-media-min("lg") {
+    align-items: center;
+  }
+
+  &__category {
+    @include hb_subheading;
+    text-transform: uppercase;
+
+    ul {
+      padding: 0;
+      margin: 0;
+    }
+
+    li {
+      display: inline-block;
+      list-style: none;
+      margin-right: 0.75rem;
+      margin-bottom: 0.3125rem;
+
+      &:last-of-type {
+        margin-right: 0;
+      }
+
+      @include colorful {
+        padding-left: 0.5rem;
+        border-left: 3px solid colorful-theme--variation(secondary);
+      }
+    }
+
+    a {
```

Comment:
If we can, we should refactor all of the elements out of the component css and rely either classes for elements or a system of utility classes. This will improve performance and if we use utility classes it will make components more flexible.